packages/vue3/src/components/Upload/MediaInput.vue

```diff
@@ -0,0 +1,87 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import BaseFileInput from './Internal/BaseFileInput.vue';
+
+const props = withDefaults(
+  defineProps<{
+    modelValue: File[]
+    limit?: number
+    disabled?: boolean
+    label: string
+    sousLabel: string
+    highLightText: string
```

Comment:
Might be better to name this one something like
```suggestion
    highlightedText: string
```

Reply:
done ✅